File: code/app/src/main/java/com/example/avtokampi_android_app/models/Uporabnik.kt
package com.example.avtokampi_android_app.models

import java.sql.Blob
import java.time.LocalDateTime


data class Uporabnik(
    val id: Int,
    val ime: String,
    val priimek: String,
    val slika: Blob,
    val telefon: String,
    val email: String,
    val geslo: String,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime,
    val pravica: Pravica
)

File: avtokampi/lib/models/Uporabnik.dart
class Uporabnik {
    int id;
    String ime;
    String priimek;
    String slika;
    String telefon;
    String email;
    String geslo;
    String createdAt;
    String updatedAt;
    int pravica;

    Uporabnik.nov();

    Uporabnik.fromUporabnik(this.id, this.ime, this.priimek, this.slika,
        this.telefon,
        this.email, this.geslo, this.createdAt, this.updatedAt, this.pravica);

    Uporabnik(this.ime, this.priimek, this.email, this.geslo);

    static Uporabnik fromJson(json) {
        Uporabnik u = new Uporabnik.nov();
        u.id = json['uporabnikId'];
        u.ime = json['ime'];
        u.priimek = json['priimek'];
        u.slika = json['slika'];
        u.telefon = json['telefon'];
        u.email = json['email'];
        u.geslo = json['geslo'];
        u.createdAt = json['createdAt'];
        u.updatedAt = json['updatedAt'];
        u.pravica = json['pravice'];
        return u;
    }
}
